src/components/CorporateCampaign/CampaignLoanRow.vue

```diff
@@ -1,126 +1,92 @@
 <template>
-	<div class="component-wrapper" ref="componentWrapper">
-		<kv-loading-spinner v-if="loadingLoans" />
+	<div class="component-wrapper">
+		<transition name="kvfade">
```

Comment:
Eventually I'd like for this loader to overlay any existing loans as it does for the grid view rather than removing and re-rendering the carousel with each change to the loan set.